### PR TITLE
[google-apps-script] Add CardService Divider and TimePicker cards

### PR DESCRIPTION
**Pull request by chrisatgoogle**
* Added Divider interface and CardService.newDivider()
* Added tests for Divider interface and CardService.newDivider()
* Added TimePicker interface and CardService.newTimePicker()
* Added tests for TimePicker interface and CardService.newTimePicker()
* Tests and lint pass:

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [CardService](https://developers.google.com/apps-script/reference/card-service/card-service?hl=en), [Divider](https://developers.google.com/apps-script/reference/card-service/divider?hl=en). [TimePicker](https://developers.google.com/apps-script/reference/card-service/time-picker?hl=en)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. Not Applicable.

```
$ npm run lint google-apps-script

> definitely-typed@0.0.3 lint
> dtslint types "google-apps-script"

$ npm test google-apps-script

> definitely-typed@0.0.3 test
> dtslint types "google-apps-script"

$
```

### CODE REVIEW

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 1]
```diff
@@ -853,6 +874,13 @@ declare namespace GoogleAppsScript {
       setWrapText(wrapText: boolean): DecoratedText;
     }
 
+    /**
+     * A horizontal divider.
+     */
+    interface Divider {
+      printJson(): string;
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.card-service.d.ts:881]: It seems `Divider#printJson` does not exists. I test in GAS.

```js
function cardService() {
  const divider = CardService.newDivider();
  Logger.log(divider.printJson());
}
```

```console
11:21:20 お知らせ 実行開始
11:21:21 エラー TypeError: divider.printJson is not a function
cardService
```

**Review by mtgto — CHANGES_REQUESTED** [review]

File: types/google-apps-script/google-apps-script.card-service.d.ts (hunk 1 above)
Comment (chrisatgoogle) [reply types/google-apps-script/google-apps-script.card-service.d.ts]: My bad. `Divider` is an empty interface and was causing a `no-empty-interface` lint error. I misunderstood how to fix it: I thought other empty interfaces were just adding `printJson()`, but it turns out they were not actually empty 🤦‍♂️. I disabled the tslint `no-empty-interface` for `Divider`, similar to the `Widget` interface and updated the PR.

**Review by mtgto — APPROVED** [review]
Thanks! 👌

**Comment by chrisatgoogle** [comment]
Ready to merge

**Pull request merged**